docs/docs/conceptual/content-sync.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Gatsby Cloud Content Sync
+---
+
+## Try Content Sync with Contentful on Gatsby Cloud
+
+To get setup quickly with a new site and have Gatsby Cloud do the heavy lifting, [deploy a new Gatsby Contentful site with just a few clicks on gatsbyjs.com](https://www.gatsbyjs.com/dashboard/deploynow?url=https://github.com/contentful/starter-gatsby-blog).
+
+## What is Content Sync?
+
+Content Sync is a Gatsby Cloud feature for improving the Preview experience for content creators.
+
+When content authors press "Open Preview" in their CMS, Content Sync routes content editors to the correct URL. It helps them understand the status of previews, so they know when their content is ready to view and otherwise if there were any build errors. It does this by displaying a loading state while the content preview builds. Once the build is complete, the editor is redirected to the correct page on the frontend of the Gatsby Preview site automatically.
```

Comment:
I really like the mental model of content sync as a "waiting room" until your content change is ready to go. I think that makes it a lot easier to visualize and conceptualize what is happening and why it's happening.

Ie something like:
It does this by first sending the content editor to a "waiting room" loading state while the content preview builds. Once the build is complete, the editor is redirected to the correct page on the frontend of the Gatsby Preview site automatically.

Reply:
Both great points, thank you 😄